runtime/drivers/druid/druid.go

```diff
@@ -43,7 +45,7 @@ func (d driver) Open(config map[string]any, shared bool, client *activity.Client
 		return nil, fmt.Errorf("require dsn to open druid connection")
 	}
 
-	db, err := sqlx.Open("avatica", dsn)
+	db, err := sqlx.Open("druid", dsn)
```

Comment:
If the `dsn` contains `/avatica-protobuf`, can we try to parse and rewrite it to the new format? (For backwards compatibility.)